alerts/openshift-container-storage-operator/StorageQuotaUtilizationThresholdReached.md

```diff
@@ -0,0 +1,22 @@
+# StorageQuotaUtilizationThresholdReached
+
+## Meaning
+
+Persistent Volume Claim (PVC) usage is alarmingly very high, indicating an imminent risk of reaching full utilization of quota. Please see the alert documentation text for an exact threshold limit.
```

Comment:
- PVC is a claim and does not have a usage, PVs have a usage
- you cannot use both "alarmingly" and "very" to describe usage, please choose one 
```suggestion
Persistent Volume (PV) usage is alarmingly high and might reach the allotted quota soon. Please see the alert documentation text for an exact threshold limit.
```

Reply:
ack